Source/WebCore/Modules/mediastream/gstreamer/GStreamerWebRTCUtils.cpp

```diff
@@ -472,6 +472,20 @@ uint32_t UniqueSSRCGenerator::generateSSRC()
     return std::numeric_limits<uint32_t>::max();
 }
 
+std::optional<int> payloadTypeForEncodingName(const char* encodingName)
+{
+    static HashMap<String, int> staticPayloadTypes = {
+        { "PCMU"_s, 0 },
+        { "PCMA"_s, 8 },
+        { "G722"_s, 9 },
+    };
+
+    auto key = String::fromLatin1(encodingName);
```

Comment:
It would be able to create a `StringView` to avoid copying the `const char*` but I don't think templates are so smart to allow calling the `contains` on that. Maybe you could turn the hash map into `HashMap<StringView, int>` ?

Reply:
Doesn't work:

```
                                                                                                                                                               
/app/webkit/WebKitBuild/Release/WTF/Headers/wtf/HashMap.h:134:10: note: candidate function not viable: no known conversion from 'const char *' to 'const WTF::HashMap<WTF::StringView, int>::KeyType' (aka 'const WTF
::StringView') for 1st argument                                                                                                                                                                                      
    bool contains(const KeyType&) const;                                                                                                                                                                             
         ^ 
```

Reply:
Ah but with AtomString it shouldn't require a copy.

Reply:
Sorry, I don't see how to micro-optimize this. Let's revisit if that really becomes a bottleneck later on. :)